lib/view_component/test_helpers.rb

```diff
@@ -162,5 +173,58 @@ def with_request_url(path)
     def build_controller(klass)
       klass.new.tap { |c| c.request = request }.extend(Rails.application.routes.url_helpers)
     end
+
+    # This is a simpler version of {Capybara::Session}.
+    #
+    # It only includes {Capybara::Node::Finders}, {Capybara::Node::Matchers},
+    # {#within} and {#within_element}. It is useful in that it does not require a
+    # session, an application or a driver, but can still use Capybara's finders
+    # and matchers on any string that contains HTML.
+    #
+    # Most of this code is shamelessly stolen from the {Capybara::Session} class
+    # in the Capybara gem (https://github.com/teamcapybara/capybara/blob/e704d00879fb1d1e1a0cc01e04c101bcd8af4a68/lib/capybara/session.rb#L38).
+    class SimpleSession
```

Comment:
Should this class be located here or somewhere else?

Reply:
I'd vote that we put it in its own class.

Reply:
Fixed in 0a3c8409.